Scripts/pod-lint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script runs a linter on every pod in the Swift SDK.
+# This must be run to ensure that every pod is ready to be published to Cocoapods.
+# The flag --allow-warnings is required for pods that have build warnings (like deprecation notices).
+
+pod lib lint IBMWatsonAssistantV1.podspec
+pod lib lint IBMWatsonConversationV1.podspec
+pod lib lint IBMWatsonDiscoveryV1.podspec
+pod lib lint IBMWatsonLanguageTranslatorV3.podspec
+pod lib lint IBMWatsonNaturalLanguageClassifierV1.podspec
+pod lib lint IBMWatsonNaturalLanguageUnderstandingV1.podspec
+pod lib lint IBMWatsonPersonalityInsightsV3.podspec
+
+pod lib lint IBMWatsonSpeechToTextV1.podspec --allow-warnings
+git reset --hard
+git clean -df
```

Comment:
These two lines are dangerous.  I think we to do this in a different, maybe more targeted way.

Reply:
Good point. I figured this would only be run on the Travis script (where it wouldn't be an issue), but running locally, this would undo all other uncommitted changes the developer may have had.